backend/authentication/tests/test_authentication.py

```diff
@@ -2,6 +2,10 @@
 from rest_framework import status
 
 
+def fail(setup_indeaa):
+    assert "True" == "False"
+
```

Comment:
do you need this?

Reply:
Whoops I do not need that I was trying to get tests running